docs/markdown/authoritative/backend-ldap.md

```diff
@@ -212,6 +211,9 @@ nsrecord:ns.example.com
 associateddomain:1.10.in-addr.arpa
 ```
 
+In `strict` mode, no other objects are required -- reverse queries that correspond to an arecord or aaaarecord of an existing object will be automagically serviced using the associateddomain entry of that object.
+
+In `simple` mode, you must then create objects for each reverse mapping:
```

Comment:
The simple mode example is immediately followed by the tree mode example from the current version of the docs ("Tree mode requires each component to be a dc element of its own...").